cassiopeia/src/main/java/edu/asu/diging/gilesecosystem/cassiopeia/core/service/impl/OCRManager.java

```diff
@@ -42,7 +42,6 @@
 @Service
 public class OCRManager implements IOCRManager {
 
-    private final Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
if you remove the logger, please also remove the imports

Reply:
Removed the unused imports.